pkg/network/infraconfigurators/masquerade_test.go

```diff
@@ -557,9 +586,9 @@ func protocols(optionalIPProtocol ...iptables.Protocol) []iptables.Protocol {
 		optionalIPProtocol...)
 }
 
-func getReservedPortList() []int {
+func getReservedPortList(isMigrationOverSockets bool) []int {
```

Comment:
This is too odd for me. The condition to have a `nil` is this boolean, so why even call this function if we know the result already?
But even if you do call this function with this off boolean, then while use `portsUsedByLiveMigration` at all (and not just return a `nil`).

Reply:
This function is being called part of the table - I simply wanted to demonstrate (so it will be clear) that this test is not using TCP sockets for migration
`newVMIMasqueradeMigrateOverSockets(namespace, vmName, getReservedPortList(!migrationOverTCP)...),`